src/v/config/configuration.cc

```diff
@@ -1470,6 +1470,25 @@ configuration::configuration()
       "performance",
       {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
       true)
+  , cloud_storage_enable_scrubbing(
+      *this,
+      "cloud_storage_enable_scrubbing",
+      "Enable scrubbing of cloud storage partitions. The scrubber validates "
+      "the integrity of data and metadata uploaded to cloud storage",
+      {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
+      true)
+  , cloud_storage_scrubbing_interval_ms(
+      *this,
+      "cloud_storage_scrubbing_interval_ms",
+      "Time interval between scrubs of the same partition",
+      {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
+      1h)
```

Comment:
Good question. This interval is applied between two scrubs of the same partition. There's also a generous jitter of 10min which tries to avoid scrubbing too many things at the same time.

To answer the question, yes (roughly). I say we go with 1h for now, but it will likely have to change to something like 6 or 12. There's some more upcoming work on scrubbing scalability and picking a good interval will be part of that.